drivers/dma/dma_stm32.h

```diff
@@ -44,14 +44,14 @@ struct dma_stm32_config {
 	struct dma_stm32_stream *streams;
 };
 
-#if defined(CONFIG_DMAMUX_STM32) || defined(CONFIG_DMA_STM32_V2)
+#if !defined(CONFIG_DMA_STM32_V1)
```

Comment:
I think there will be a problem with h7 socs. The dma controller is v1 but is has a dmamux. So it should start with 1

Reply:
checking...